mantidproject/mantid
# Muon analysis raw plot speed

**Description of work.**

The loading speed of muon analysis was slower than previous version as it was creating 16 tiled plots.

So we have disabled the raw plotting.

**To test:**


Open muon analysis
Load some data
The load speed should be faster than without this PR and a normal work flow should be fine

Do the same for the frequency domain analysis GUI and a normal work flow should be fine

Fixes #32379



*This does not require release notes* because it fixes a speed issue of a new feature.




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
